DataFormats/TrackerRecHit2D/src/classes_def.xml

```diff
@@ -163,4 +124,37 @@
   <class name="reco::ClusterRemovalInfo::SiPixelClusterRefProd" />
   <class name="reco::ClusterRemovalInfo::SiStripClusterRefProd" />
   <class name="edm::Wrapper<reco::ClusterRemovalInfo>" />
+
+  // fastsim containers
+  
+  <class name="FastTrackerRecHit" ClassVersion="10">
```

Comment:
I had in mind iorules or converters if need to read old info.
On Aug 27, 2015 11:03 AM, "lveldere" <notifications@github.com> wrote:

> In DataFormats/TrackerRecHit2D/src/classes_def.xml
> <https://github.com/cms-sw/cmssw/pull/10987#discussion_r38113061>:
>
> > @@ -163,4 +124,37 @@
> >    <class name="reco::ClusterRemovalInfo::SiPixelClusterRefProd" />
> >    <class name="reco::ClusterRemovalInfo::SiStripClusterRefProd" />
> >    <class name="edm::Wrapper<reco::ClusterRemovalInfo>" />
> > +
> > +  // fastsim containers
> > +
> > +  <class name="FastTrackerRecHit" ClassVersion="10">
>
> hi slava
>
> I would be very surprised if there were any interest in running with 76 on
> samples with older releases. But I could put the old classes_def lines back
> if that is desirable and technically possible.
> (I have no clue how CMSSW will react having lines for classes that no
> longer exist)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cms-sw/cmssw/pull/10987/files#r38113061>.
>